oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthenticationCallbackHandler.java

```diff
@@ -343,6 +345,15 @@ public APIGatewayProxyResponseEvent handleRequest(
                 userInfoStorageService.addAuthenticationUserInfoData(
                         userInfo.getSubject().getValue(), userInfo);
 
+                OrchSessionItem orchSession =
+                        orchSessionService
+                                .getSession(sessionCookiesIds.getSessionId())
+                                .orElseThrow(
+                                        () ->
+                                                new AuthenticationCallbackException(
+                                                        "Orchestration user session not found"));
```

Comment:
Could we move this session code next to the existing session code?

Reply:
Done, also changed the exception message for clarity